### PR TITLE
Added Recommendation ID filtering to Compliance data.

### PR DESCRIPTION
**Pull request by alannix-lw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] Ready
~~In Hold - Depends on https://github.com/demisto/dockerfiles/pull/3416~~ @bakatzir - I merged it. Eli - You can review.

## Related Issues
fixes: link to the issue

## Description
This commit adds the ability to filter returned compliance data via command rather than in a playbook.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Lacework/Integrations/Lacework/Lacework.py [hunk 1]
```diff
@@ -74,6 +74,13 @@ def format_compliance_data(compliance_data):
 
         compliance_data = compliance_data['data'][0]
 
+        # If the user wants to filter on a recommendation ID
+        if rec_id:
+            # Iterate through all recommendations, removing irrelevant ones
+            for recommendation in compliance_data["recommendations"][:]:
+                if rec_id != recommendation["REC_ID"]:
+                    compliance_data["recommendations"].remove(recommendation)
+
```
Comment (esharf) [inline Packs/Lacework/Integrations/Lacework/Lacework.py:83]: not about it but it looks that `rec_id` can be easily converted to an array supporting parameter.
(not familiar with the use-case but looks like a quick win)

Comment (alannix-lw) [reply Packs/Lacework/Integrations/Lacework/Lacework.py:83]: Thanks for the suggestion, Eli!  The initial use case for this was to simply return one result for playbooks, but checking against a list of recommendations still allows for that use case, and makes the command more extensible.  I'll push a revision in just a few minutes.

File: Packs/Lacework/Integrations/Lacework/Lacework.py [hunk 2]
```diff
@@ -121,14 +129,15 @@ def get_azure_compliance_assessment():
 
     tenant_id = demisto.args().get('tenant_id')
     subscription_id = demisto.args().get('subscription_id')
+    rec_id = demisto.args().get('rec_id')
```
Comment (esharf) [inline Packs/Lacework/Integrations/Lacework/Lacework.py:132]:
```suggestion
    rec_id = argToList(demisto.args().get('rec_id'))
```

**Review by esharf — CHANGES_REQUESTED** [review]

File: Packs/Lacework/Integrations/Lacework/Lacework.py (hunk 2 above)
Comment (alannix-lw) [reply Packs/Lacework/Integrations/Lacework/Lacework.py]: Logic updated to leverage the argToList function.

**Review by esharf — APPROVED** [review]

**Pull request merged**